ublox_gps/src/node.cpp

```diff
@@ -489,6 +492,9 @@ void UbloxNode::getRosParams() {
   if (getRosBoolean(this, "publish.aid.hui")) {
     aid_hui_pub_ = this->create_publisher<ublox_msgs::msg::AidHUI>("aidhui", 1);
   }
+  if (getRosBoolean(this, "publish.nmea")) {
+    nmea_pub_ = this->create_publisher<nmea_msgs::msg::Sentence>("nmea", 20);
```

Comment:
Any particular reason to have a deeper depth here?  Otherwise, I'd say just leave this at 1 like the rest of the publishers.

Reply:
There are a number of NMEA messages that get parsed in each ublox update, and they all get processed and queued one right after the other. The callback is called faster than the ROS publisher's queue clears, resulting in messages getting dropped if the queue depth is only 1.

Reply:
All right, that is a good explanation.  Can you please put a small comment here indicating that?